concent_api/conductor/tasks.py

```diff
@@ -17,6 +18,7 @@
 
 @shared_task
 @provides_concent_feature('conductor-worker')
+@transaction.atomic(using='storage')
```

Comment:
@pawelkisielewicz I think not in this case - those values are database names from settings, so if someone will someday change the name of the database, it will require updating enum too, which I think is not matching enum purpose.

Reply:
Yeah. `atomic()` is a function from Django. We would have to convert it to string when calling it anyway so there's not much point in creating an enum.